### PR TITLE
Formatter: proper multiline array elements indent

### PR DESCRIPTION
**Pull request by VladRassokhin**

### CODE REVIEW

File: src/test/resources/testData/format/indents_res.ps1 [hunk 1]
```diff
@@ -28,4 +28,14 @@ function c()
...
+function d()
+{
+    $arr = @(
+        1
+        "2"
+        "3=$( 3 )"
+        4, 5
+        6
+    )
 }
```
Comment (kevinholtkamp) [inline src/test/resources/testData/format/indents_res.ps1:41]: Shouldn't the closing curly brace be kept?

**Comment by kevinholtkamp** [comment]
Accidental review, sorry.
Really in favor of this change!

**Comment by Omico** [comment]
A huge thanks for this PR! You are my lifesaver!

**Comment by rwidmark** [comment]
@ant-druha is this repo dead?

**Review by ForNeVeR — APPROVED** [review]
Thank you!

Sorry it took some time to get this merged, but we are getting back on track.

**Pull request merged**